recipe/meta.yaml

```diff
@@ -7,9 +7,12 @@ package:
 source:
   url: https://pypi.io/packages/source/C/Cython/Cython-{{ version }}.tar.gz
   sha256: 6a0d31452f0245daacb14c979c77e093eb1a546c760816b5eed0047686baad8e
+  patches:
+    # part of next release
+    - patches/pypy37-eval.patch
```

Comment:
Where does this patch come from? Is there an upstream PR or commit with this change?

Reply:
I submitted cython/cython#4236 just now.

Reply:
Thanks Matti! 😄